housekeeping/src/main/java/com/hotels/housekeeping/conf/Housekeeping.java

```diff
@@ -39,6 +41,9 @@
   @Min(1)
   private int fetchLegacyReplicaPathPageSize = DEFAULT_FETCH_LEGACY_REPLICA_PATH_PAGE_SIZE;
 
+  @Min(1)
+  private int cleanupThreads = DEFAULT_NUMBER_OF_CLEANUP_THREADS;
```

Comment:
Is this configurable? If so, it should probably go in to the README?

Reply:
it is in the readme in the table of properties to set: `cleanup-threads`

Reply:
ah my mistake, apologies